docs/guides/setup/configuring_metadata_stores/how_to_configure_a_metricsstore.md

```diff
@@ -1,75 +1,110 @@
+
+Here is an example yaml config for adding a StoreMetricsAction to the `taxi_data` dataset:
 
 ```yaml
-validation_operators:
+action_list:
   # ...
-  action_list_operator:
-    class_name: ActionListValidationOperator
-    action_list:
-      # ...
-      - name: store_metrics
-        action:
-          class_name: StoreMetricsAction
-          target_store_name: metrics_store  # Keep the space before this hash so it's not read as the name. This should match the name of the store configured above
-          # Note that the syntax for selecting requested metrics will change in a future release
-          requested_metrics:
-            "*":  # The asterisk here matches *any* Expectation Suite name
-              # use the 'kwargs' key to request metrics that are defined by kwargs,
-              # for example because they are defined only for a particular column
-              # - column:
-              #     Age:
-              #       - expect_column_min_to_be_between.result.observed_value
-              - statistics.evaluated_expectations
-              - statistics.successful_expectations
+  - name: store_metrics
+    action:
+      class_name: StoreMetricsAction
+      target_store_name: metric_store  # This should match the name of the store configured above
+      requested_metrics:
+        public.taxi_data.warning:  # expectation suite name
+          - statistics.evaluated_expectations
+          - statistics.successful_expectations
+          - statistics.success_percent
+          - statistics.unsuccessful_expectations
+        "*":  # wildcard to match any expectation suite
+          - column:
+            passenger_count:
+              - expect_column_values_to_not_be_null.result.element_count
+              - expect_column_values_to_not_be_null.result.partial_unexpected_list
 ```
 
-The `StoreMetricsValidationAction` processes an `ExpectationValidationResult` and stores Metrics to a configured Store.
-Now, when your operator is executed, the requested metrics will be available in your database!
-
+### Test your MetricStore and StoreMetricsAction
 
+To test your `StoreMetricsAction`, run your checkpoint from your code or the CLI:
 ```python
-context.run_validation_operator('action_list_operator', (batch_kwargs, expectation_suite_name))
+import great_expectations as ge
+context = ge.get_context()
+checkpoint_name = "your checkpoint name here"
+context.run_checkpoint(checkpoint_name=checkpoint_name)
 ```
+```bash
+$ great_expectations checkpoint run <your checkpoint name>
+```
+
+### Summary
+The `StoreMetricsValidationAction` processes an `ExpectationValidationResult` and stores Metrics to a configured Store.
+Now, after your Checkpoint is run, the requested metrics will be available in your database!
 
 :::note
 To discuss with the Great Expectations community, please visit this topic in our community discussion forum: https://discuss.greatexpectations.io/t/ge-with-databricks-delta/82/3
```

Comment:
It looks like this needs to be updated / removed.

Reply:
nice catch, thanks!